codecov.yml

```diff
@@ -2,11 +2,17 @@ coverage:
   status:
     project:
       default:
-        target: 90%
+        target: auto
```

Comment:
https://docs.codecov.com/docs/quick-start#tips-and-tricks

`status.project.default.target: auto` compares coverage to the previous base commit

![image](https://github.com/TanStack/query/assets/61593290/aff871a6-e174-45d5-95c4-c904c1d2f732)

Reply:
I think this(https://docs.codecov.com/docs/common-recipe-list#isolate-my-coverage) is good to isolate target coverage of each packages too.